### ISSUE TITLE
Add custom columns also in conversation tab

### ISSUE THREAD

**Issue by micz** [micz/ColumnsWizard#48]
Add custom columns also in conversation tab.
Add columns picker menu also in conversation tab.

**Comment by paulcrown** [comment]
I agree adding the custom columns in the account search messaged dialog would be helpful.
